source/tools/index.rst

```diff
@@ -58,6 +58,8 @@ This tool is available on GitHub: |https://github.com/intel/iasimage|
 
    <a href="https://github.com/intel/iasimage" target="_blank">https://github.com/intel/iasimage</a>
 
+.. note:: Currently ``iasimage`` is only supported by Python 3 version.

```

Comment:
should we support python2 too ?  It is not convenient to have both python2/3 installed.

Reply:
This bug in Python 2 must be fixed first before removing this note.